lib/ts/utils.ts

```diff
@@ -205,3 +206,29 @@ export function validateTheStructureOfUserInput(
         });
     }
 }
+
+export async function getAPIVersionFromFileIfExists(): Promise<string | null> {
```

Comment:
Please use `undefined` instead of `null`

Reply:
done